instance.cc

```diff
@@ -2955,28 +3015,34 @@ Heightfield PGInstance::getHeightField(float bx, float bz) {
 
     // postprocess height
     {
-        std::vector<unsigned char> seenBiomes;
-        for (auto kv : biomeCounts)
+        std::vector<uint8_t> seenBiomes;
+        for (uint8_t b = 0; b < biomeCounts.size(); b++)
```

Comment:
Technically this isn't correct, `biomeCounts.size()` returns `size_t`.

I would make the narrowing cast explicit in the below code rather than hidden in this loop.